tests/test_path_follow_2d.h

```diff
@@ -81,15 +81,15 @@ TEST_CASE("[PathFollow2D] Sampling with unit offset") {
 }
 
 TEST_CASE("[PathFollow2D] Sampling with offset") {
-	const Ref<Curve2D> &curve = memnew(Curve2D());
+	Ref<Curve2D> curve = memnew(Curve2D);
 	curve->add_point(Vector2(0, 0));
 	curve->add_point(Vector2(100, 0));
 	curve->add_point(Vector2(100, 100));
 	curve->add_point(Vector2(0, 100));
 	curve->add_point(Vector2(0, 0));
-	const Path2D *path = memnew(Path2D);
+	Path2D *path = memnew(Path2D);
```

Comment:
The `PathFollow2D` tests crash on instantiating `Path2D` since it's a `CanvasItem` and that constructor calls `RenderingServer::get_singleton()->canvas_item_create()` but that singleton is null since it seems the `RenderingServer` is never initialized for tests. Not sure what to do here.